src/Config/CS.props

```diff
@@ -1,5 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>
+      None
+    </ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>
```

Comment:
Architecture mismatch warnings because of msil(anycpu) vs amd64 (x64) 
 maybe we can just have one : anycpu

Reply:
In the past we needed x64 for CEF - now that we've gotten rid of it I think you're right.

Reply:
I will remove x64 when I will move the rest of the projects to sdk style